File: cpp/Program/Vehicle.cpp
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Vehicle
{
private:
    string platNomor;
    string merk;
    string tahunProduksi;
    string warna;

public:

    Vehicle(){

    }

    Vehicle(string platNomor){
        this->platNomor = platNomor;
        merk = "";
        tahunProduksi = "";
        warna = "";
    }

    Vehicle(string platNomor, string merk){
        this->platNomor = platNomor;
        this->merk = merk;
        tahunProduksi = "";
        warna = "";
    }

    Vehicle(string platNomor, string merk, string tahunProduksi){
        this->platNomor = platNomor;
        this->merk = merk;
        this->tahunProduksi = tahunProduksi;
        warna = "";
    }

    Vehicle(string platNomor, string merk, string tahunProduksi, string warna){
        this->platNomor = platNomor;
        this->merk = merk;
        this->tahunProduksi = tahunProduksi;
        this->warna = warna;
    }

    string GetPlatNomor() {
    return platNomor;
    }

    void SetPlatNomor(string platNomor) {
    this->platNomor = platNomor;
    }

    string GetMerk() {
        return merk;
    }

    void SetMerk(string merk) {
        this->merk = merk;
    }

    string GetTahunProduksi() {
        return tahunProduksi;
    }

    void SetTahunProduksi(string tahunProduksi) {
        this->tahunProduksi = tahunProduksi;
    }

    string GetWarna() {
        return warna;
    }

    void SetWarna(string warna) {
        this->warna = warna;
    }

     ~Vehicle()
    {


    }

    
};

File: python/Program/Vehicle.py
class Vehicle:
    __platNomor = ""
    __merk = ""
    __tahunProduksi = ""
    __warna = ""
    
    def __init__(self, platNomor = "", merk = "", tahunProduksi = "", warna = ""):
        self.__platNomor = platNomor
        self.__merk = merk 
        self.__tahunProduksi = tahunProduksi
        self.__warna = warna

    def get_platNomor(self):
        return self.__platNomor

    def set_platNomor(self, value):
        self.__platNomor = value

    def get_merk(self):
        return self.__merk

    def set_merk(self, value):
        self.__merk = value

    def get_tahunProduksi(self):
        return self.__tahunProduksi
    
    def set_tahunProduksi(self, value):
        self.__tahunProduksi = value

    def get_warna(self):
        return self.__warna

    def set_warna(self, value):
        self.__warna = value

    